I think the function combineTransformsToTRS in Geometry3d.ts should be modified, the function applied to the quaternions near line 154 should be changed to premultiply, so that the final version would be:

```
const q1 = new Quaternion().setFromEuler(rotation);
const q2 = new Quaternion().setFromEuler(euler);
q1.premultiply(q2);
rotation.setFromQuaternion(q1, "XYZ");
```

While all of the unit tests succeed with the original code ( https://github.com/IIIF-Commons/manifesto-3d/pull/29/commits/23d2bf1b7e3d54525bfdd54aaf01c914b1e1c91f ), the test that combined two RotateTransforms, the tests with 'combines transform rotate -> rotate' , combine two RotateTransforms both about the x-axis, in this case the quaternion multiplication is independent of order (so multiply will give the same result are premultiply )

I prepared unit tests that combine a RotateTransform along the x axis with with along the y axis, in detail the transform tested is:

```
transform : [
    {
        type: "RotateTransform",
        y : 45
    },
    {
        type: "RotateTransform",
        x : 60
    }
]

```
To complete the unit test I am asserting that this combination of rotations is geometrically equal to :

```
transform : [
    {
        type: "RotateTransform",
        x : 60,
        y : 45
    }
]
```

I recognize that this is based on an interpretation of combining rotations inside a transform list, and that there may be a different consistent interpretation based on redefining the X and Y axis within the transform list in the same way that intrinsic axes are using within a single Euler rotation; but I can't see that explained in the three-js documentation.

However, making adding this unit test to the set for both the 'combineTransformsToMatrix' test suite and the 'combineTransformsToTRS'  under the description 'combines transform rotate -> rotate (2)' has results:
-- with current code , at commit   , the new unit test in the combineTransformsToMatrix suite, which is not sensitive to quaternion multiplication, succeeds;  while in the combineTransformsToTRS suite it fails.

-- when the quaternion calculation is changed to .premultiply, the new unit test succeeds in both suites; all other unit tests succeed.

I have not changed the code in Gerometry3d.ts, so at this commit on test should fail:
```
 1) iiif-3d tests
       Geometry3d: combineTransformsToTRS
         combines transform rotate -> rotate (2):
     AssertionError: expected 1.1831996401397158 to be close to 1.0471975511965976 +/- 0.000001
      at Context.<anonymous> (test/tests_3d/core_tests/Geometry3d.js:589:34)
      at process.processImmediate (node:internal/timers:478:21)

```

--- a/src/Geometry3d.ts
+++ b/src/Geometry3d.ts
@@ -151,7 +151,7 @@ export function combineTransformsToTRS(transforms: Transform[]): TransformSet {
             const euler = eulerFromRotateTransform(transform as RotateTransform);
             const q1 = new Quaternion().setFromEuler(rotation);
             const q2 = new Quaternion().setFromEuler(euler);
-            q1.multiply(q2);
+            q1.premultiply(q2);
             rotation.setFromQuaternion(q1, "XYZ");
             translation.applyEuler(euler);
         } else if (transform.isScaleTransform) {

